sdk/eventhub/Azure.Messaging.EventHubs/CHANGELOG.md

```diff
@@ -1,15 +1,11 @@
 # Release History
 
-## 5.6.0-beta.1 (Unreleased)
+## 5.5.1 (2021-06-09)
```

Comment:
Should this be a minor version bump since we are changing the API surface?

Reply:
Good point.   It probably should.